hc/api/tests/test_pause.py

```diff
@@ -1,6 +1,8 @@
 from hc.api.models import Check
 from hc.test import BaseTestCase
 
```

Comment:
To comply with pep8, consider making this import on the first line and remove the blank white space